src/units.rs

```diff
@@ -0,0 +1,57 @@
+/// This structure represent all the stats that a unit can have.
+struct Stats {
+    /// he distance the model can Advance Move in inches per turn.
+    advance_rate: usize,
+    /// The distance the model can March Move in inches per turn.
+    march_rate: usize,
+    /// Represents the model’s overall morale, bravery and a unit’s ability to quickly reorganise during battle.
+    discipline: usize,
+    /// When the model loses this many Health Points, it is removed as a casualty
+    health_point: isize,
+    /// How well the model avoids being hit in melee. A high Defensive Skill may also represent models with equipment useful for parrying melee attacks, such as shields. 
+    defense: usize,
+    /// How easily the model withstands blows.
+    resilience: usize,
+    /// The Armour of the model. This may represent particularly tough scaly hides as well as worn pieces of Armour.
+    armour: usize,
+    /// A special protection for the model, the final defense after the model’s Armour. This may represent a model with magical defenses or regenerative abilities.
+    aegis: usize,
+    /// The number of Melee Attacks the model has. This maybe higher due to sheer attacking speed and skill, equipment such as dual wielded blades, or the size to potentially hit many opponents with a single blow.
+    attack: usize,
+    /// How good the model is at scoring hits in melee.
+    offensive: usize,
+    /// How easily the model can wound enemy models in melee. This may represent not just sheer physical strength, but also well-equipped models with weapons such as halberds or heavy two handed weaponry.
```

Comment:
```suggestion
    /// How easily the model can wound enemy models in melee.
```
The last part isn't really useful